### PR TITLE
Fix HID.requestDevice macro in WebHID_API

### PR DESCRIPTION
**Pull request by AnilSeervi**

### CODE REVIEW

File: files/en-us/web/api/webhid_api/index.html [hunk 1]
```diff
@@ -32,7 +32,7 @@ <h2 id="Interfaces">Interfaces</h2>
 
 <h2 id="Examples">Examples</h2>
 
-<p>You can connect to a device with {{domxref("HID.requestDevice()")}}. In this case, we select from all the available devices.</p>
+<p>You can connect to a device with {{domxref("HID.requestDevice","requestDevice()")}} method. In this case, we select from all the available devices.</p>
```
Comment (teoli2003) [inline files/en-us/web/api/webhid_api/index.html:35]:
```suggestion
<p>You can connect to a device with the {{domxref("HID.requestDevice","requestDevice()")}} method. In this case, we select from all the available devices.</p>
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
A tiny fix is needed here

File: files/en-us/web/api/webhid_api/index.html (hunk 1 above)
Comment (AnilSeervi) [reply files/en-us/web/api/webhid_api/index.html]: Sure. On it.

**Review by teoli2003 — APPROVED** [review]

**Comment by teoli2003** [comment]
Thanks!

**Pull request merged**